addons/l10n_es_pos/models/pos_config.py

```diff
@@ -0,0 +1,87 @@
+from odoo import _, api, fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    is_spanish = fields.Boolean(string="Company located in Spain", compute="_is_company_spanish")
+
+    def _is_company_spanish(self):
+        for pos in self:
+            pos.is_spanish = pos.company_id.country_id.code == "ES"
+
+    l10n_es_simplified_invoice_limit = fields.Float(
+        string="Sim.Inv limit amount",
+        digits="Account",
+        help="Over this amount is not legally possible to create a simplified invoice",
+        default=400, 
+    )
+    l10n_es_simplified_invoice_sequence_id = fields.Many2one(
+        "ir.sequence",
+        string="Simplified Invoice IDs Sequence",
+        help="Autogenerate for each POS created",
+        copy=False,
+        readonly=True,
+    )
+
+    def get_l10n_es_simplified_invoice_number(self):
+        self.ensure_one()   
+        return self.l10n_es_simplified_invoice_sequence_id._get_current_sequence().number_next_actual
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        records = super().create(vals_list)
+        records._set_sequence()
+        return records
+
+    def _set_sequence(self):
+        for record in self:
+            if record.is_spanish:
+                record.l10n_es_simplified_invoice_sequence_id = record._create_sequence().id
```

Comment:
Can you put the create sequence inside this function ? 
We don't need this call since it's never used anywhere else